docs/articles/configs/toolchains.md

```diff
@@ -339,7 +339,7 @@ And FINALLY build Mono Runtime with Web Assembly support:
 Before you run the benchmarks, you need to make sure that following two file exists:
 
 ```cmd
-runtime/src/mono/wasm/runtime-test.js
```

Comment:
I think we actually need to delete this; the documentation is old. We no longer directly specify the path to this file.